lang/RStudio/RStudio.def

```diff
@@ -0,0 +1,45 @@
+Bootstrap: library
+From: debian:9
+
+%environment
+    LC_ALL="C"
+    export LC_ALL
+
+%post
+    apt-get update
+    apt-get -y install systemd systemd-sysv gdebi-core procps libssl1.1 ed wget curl libqt5webkit5 libqt5core5a
+    apt-get -y install r-base r-base-dev
+
+    curl https://download1.rstudio.org/rstudio-xenial-1.1.456-amd64.deb > /rstudio-1.1.456-amd64.deb
+    apt-get -y install /rstudio-1.1.456-amd64.deb
+
+    wget -O /rstudio-server-stretch-1.1.456-amd64.deb \
+      https://download2.rstudio.org/rstudio-server-stretch-1.1.456-amd64.deb
+    gdebi -n /rstudio-server-stretch-1.1.456-amd64.deb
+
+    # Setup the "general" CRAN repo
+    echo 'local({
+ r <- getOption("repos")
+ r["CRAN"] = "https://cran.us.r-project.org"
+ options(repos = r)
+})' >> /etc/R/Rprofile.site
```

Comment:
Check this indentation here

Reply:
It's right when it ends up in the file. ;)